<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QWidget" name="histWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <layout class="QVBoxLayout" name="histLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="imageWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="imageLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Acquisition</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="acqImageButton">
           <property name="text">
            <string>Acq. Images</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="acqmodeCombo"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>ICT</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="acqChargeButton">
           <property name="text">
            <string>Acq. Charge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="chargeLabel">
           <property name="text">
            <string>Charge: 0 pC</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Camera</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QSpinBox" name="exposureInput">
             <property name="minimum">
              <number>500</number>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="singleStep">
              <number>500</number>
             </property>
             <property name="value">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="gainLabel">
             <property name="text">
              <string>Gain</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="gainInput">
             <property name="maximum">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="cameraCombo"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Exposure</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Colormap</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="colormapCombo"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Background</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="setButton">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="subtractButton">
           <property name="text">
            <string>Subtract</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Saving</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="descriptionEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="scanButton">
           <property name="text">
            <string>Scan</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="imagescanButton">
           <property name="text">
            <string>Scan + Images</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save Image</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Filter Wheel</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="filterCombo"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Image Analysis</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="7" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>pixelsum</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="pixelSumAvg">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="avgLabel">
             <property name="text">
              <string>Avg.</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="xcInstant">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QLabel" name="pixelSumInstant">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Instant.</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="ellipseButton">
             <property name="text">
              <string>Ellipse</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="ycAvg">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Yc</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="ycInstant">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="yrmsInstant">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Xrms</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Yrms</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="xrmsInstant">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="xcAvg">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="xrmsAvg">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="analyzeButton">
             <property name="text">
              <string>Analyze</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="roiButton">
             <property name="text">
              <string>ROI</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="resetButton">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Xc</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="yrmsAvg">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>deltax</string>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QLabel" name="deltayInstant">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Averaged Shots</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>deltay</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QLabel" name="deltaxInstant">
             <property name="text">
              <string>0um</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="shotsInput">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>correctedsum</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="correctedSumAvg">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QLabel" name="correctedSumInstant">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
